geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/membership/GMSJoinLeave.java

```diff
@@ -1412,7 +1413,7 @@ void setJoinResponseMessage(JoinResponseMessage<ID> jrm) {
     joinResponse[0] = jrm;
   }
 
-  void processMessage(FindCoordinatorRequest<ID> req) {
+  void processFindCoordinatorMessage(FindCoordinatorRequest<ID> req) {
```

Comment:
should be `processFindCoordinatorRequestMessage()`

Reply:
fixed